types/ws/ws-tests.js

```diff
@@ -0,0 +1,354 @@
+"use strict";
```

Comment:
It would seem the CI process failed be cause this file was included. It looks like a compiled version of the test file, which should not be included in the code here. Could you remove this file from being tracked?